### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by ebell495**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -1,12 +1,13 @@
 cmake_minimum_required(VERSION 3.20)
```
Comment (xansec) [inline src/CMakeLists.txt:1]: Can you comment on why these changes were necessary?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: src/CMakeLists.txt (hunk 1 above)
Comment (ebell495) [reply src/CMakeLists.txt:1]: This was to fix a linking issue that I was having with the math library. There was also an issue opened by someone on the original repository AnonymousAAArdvark/qi#16

**Review by xansec — APPROVED** [review]
Gotcha, thanks for the info! LGTM!
